osmaxx-py/excerptconverter/baseexcerptconverter/base_excerpt_converter.py

```diff
@@ -0,0 +1,89 @@
+import abc as abstract_base_class
+import stored_messages
+
+from celery import shared_task
+
+from django.contrib import messages
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+
+
+class BaseExcerptConverter(metaclass=abstract_base_class.ABCMeta):
+    available_converters = []
+    name = 'base'
+    export_formats = {}
+    export_options = {}
+    steps_total = 0
```

Comment:
Only `available_converters` is needed in the abstract base class. The other class fields here should be defined in concrete derived classes, only, _not_ in the abstract base class.

To document what the base class requires of its concrete derivatives, a [Python Docstring](https://www.python.org/dev/peps/pep-0257/) is the way to go, rather than dead code.